packages/configuration-validation/package.json

```diff
@@ -32,17 +34,38 @@
   },
   "homepage": "https://github.com/okta/okta-oidc-js#readme",
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.1.1",
-    "babel-preset-env": "^1.7.0",
+    "@babel/core": "^7.8.0",
+    "@babel/plugin-transform-typescript": "^7.11.0",
+    "@babel/preset-env": "^7.8.2",
+    "@babel/preset-typescript": "^7.10.4",
+    "@types/jest": "^25.2.3",
+    "@types/json-schema": "^7.0.5",
+    "@types/minimatch": "^3.0.3",
+    "@types/node": "^14.6.1",
+    "@typescript-eslint/eslint-plugin": "^2.34.0",
+    "@typescript-eslint/parser": "^2.34.0",
+    "babel-jest": "^26.3.0",
     "deep-extend": "^0.6.0",
-    "eslint": "^4.7.1",
-    "jest": "^23.6.0"
+    "eslint": "^7.7.0",
+    "jest": "^26.4.2",
+    "rimraf": "^3.0.2",
+    "typescript": "^4.0.2"
   },
   "jest": {
-    "testEnvironment": "jsdom"
+    "moduleNameMapper": {
+      "^@okta/configuration-validation$": "<rootDir>/src/lib"
+    },
+    "restoreMocks": true,
+    "testEnvironment": "jsdom",
+    "testMatch": [
+      "**/test/*.{js,ts}"
+    ],
+    "roots": [
+      "test"
+    ]
   },
   "dependencies": {
+    "@okta/okta-auth-js": "^4.0.0",
```

Comment:
Looks like it only depends on types from `auth-js`, should it belongs to `dependencies` or `devDependenies`?

Reply:
never mind, looks like the module also exports new types based on the imported types.